src/main/java/com/elixr/poc/data/User.java

```diff
@@ -8,7 +8,7 @@
 import java.util.UUID;
 
 /**
```

Comment:
Except the same comment line coming one line down compared to previous state, there is no change in this file. You can roll back this back to previous state unless someone has commented on it

Reply:
Yes the commented lines are the changes made by others, to remove the conflicts I just add that line.